exchanges/execute/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@urql/exchange-execute",
+  "version": "1.0.0",
+  "description": "An exchange for executing queries against a local schema in urql",
+  "sideEffects": false,
+  "homepage": "https://formidable.com/open-source/urql/docs/",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/urql.git",
+    "directory": "exchanges/execute"
+  },
+  "keywords": [
+    "urql",
+    "exchange",
+    "execute",
+    "executable schema",
+    "formidablelabs",
+    "exchanges"
+  ],
+  "main": "dist/urql-exchange-execute.cjs.js",
```

Rebase your branch, we altered the conventions we use to be

```
main: dist/urql-exchange-execute
module: dist/urql-exchange-execute.mjs
```

This has to be done for the exports map as well but these need the path "./" included for node 13

---

@JoviDeCroock  Like this? 

```suggestion
  "main": "./dist/urql-exchange-execute",
  "module": "./dist/urql-exchange-execute.mjs",
```

---

Done.